jcl/src/jdk.management/share/classes/com/ibm/lang/management/OperatingSystemMXBean.java

```diff
@@ -36,29 +35,51 @@
  * ...
  * }
  * </pre></td></tr></table>
+ * <br>
+ *
+ * The following methods depend on certain information that is not available on z/OS;
+ * their return values are listed here:
+ * <table border="1">
+ * <tr>
+ * <th align="left">Method</th>
+ * <th align="left">Return Value</th>
+ * </tr>
+ * <tr>
+ * <td>{@link getCommittedVirtualMemorySize()}</td>
+ * <td>{@code -1}</td>
+ * </tr>
+ * <tr>
+ * <td>{@link getFreePhysicalMemorySize()}</td>
```

Comment:
Pls show the non-deprecated method as well. For getSystemCpuLoad() too.

Reply:
Added `getCpuLoad()` and `getFreeMemorySize()` to this table for 14+.